lib/calendarium-romanum/temporale.rb

```diff
@@ -129,7 +129,13 @@ def nativity(year=nil)
     end
 
     def holy_family(year=nil)
-      sunday_after(nativity(year))
+      year ||= @year
+      xmas = nativity(year)
+      if xmas.sunday?
+        return Date.new(year, 12, 30) # and only 1 reading before Gospel
```

Comment:
Please remove the comment - lectionary is (and will always be) out of scope of this gem, so it's of no use here.